contracts/HumaniqToken.sol

```diff
@@ -63,76 +70,80 @@ contract HumaniqToken is StandardToken, SafeMath {
     function issueTokens(address _for, uint tokenCount)
         external
         payable
-        isCrowdfundingContract
+        onlyMinter
         returns (bool)
     {
         if (tokenCount == 0) {
             return false;
         }
-        
-        if (icoFinished && add(totalSupply, tokenCount) > maxTotalSupply) {
-          throw;
+
+        if (add(totalSupply, tokenCount) > maxTotalSupply) {
+            throw;
         }
-        
+
         totalSupply = add(totalSupply, tokenCount);
         balances[_for] = add(balances[_for], tokenCount);
         Issuance(_for, tokenCount);
         return true;
     }
 
-    function transfer(address _to, uint256 _value)
-        unlocked
-        returns (bool success)
-    {
-        return super.transfer(_to, _value);
-    }
-
-    function transferFrom(address _from, address _to, uint256 _value)
-        unlocked
-        returns (bool success)
-    {
-        return super.transferFrom(_from, _to, _value);
-    }
-
     /// @dev Function to change address that is allowed to do emission.
     /// @param newAddress Address of new emission contract.
-    function changeEmissionContractAddress(address newAddress)
-        external
+    function changeMinter(address newAddress)
+        public
         onlyFounder
         returns (bool)
     {
-        emissionContractAddress = newAddress;
+        minter = newAddress;
+
+        // Allow emission contract to distribute tokens minted during ICO stage
+        allowed[allocationAddressICO][minter] = ICOSupply;
```

Comment:
Doesn't really matter. Even if allowed value exceeds remaining balance, transferFrom function doesn't allow spend more money than exist.
Agree with the second point.